src/main/java/com/hubspot/imap/protocol/command/SimpleStringCommand.java

```diff
@@ -0,0 +1,15 @@
+package com.hubspot.imap.protocol.command;
+
+public class SimpleStringCommand extends BaseImapCommand {
```

Comment:
Probably a better name but functionally equivalent to https://github.com/HubSpot/NioImapClient/blob/master/src/main/java/com/hubspot/imap/protocol/command/StringLiteralCommand.java

Reply:
ahh ya I thought the `StringLiteralCommand` send a size param (because that is what a `StringLiteral` is in IMAP) will rename